openwhyd/openwhyd
# feat: eject "solo" version of Openwhyd

## Goal

Prepare a stripped-down version of Openwhyd, with fewer files, to reduce cognitive load while refactoring it during our Devoxx university talk.

## Changes

Only keep features that allow a single user to manage and play their collection of tracks. I.e. delete all social features.

<img width="977" alt="image" src="https://user-images.githubusercontent.com/531781/153713929-6bed94dc-2610-4289-b3b3-0f19753c6c94.png">

Not safe for public use in production.

## How to test

1. Clone/checkout this PR's branch
2. Make sure that Node.js 16 is installed (recommendation: use [nvm](https://github.com/nvm-sh/nvm) to install it, so you can easily switch from one version to another, if needed)
3. Start with docker-compose and run automated tests: https://github.com/openwhyd/openwhyd/blob/master/docs/INSTALL.md#run-automated-tests
